tests/draft-next/unevaluatedProperties.json

```diff
@@ -1516,5 +1516,27 @@
                 "valid": false
             }
         ]
+    },
+    {
+        "description": "unevaluatedProperties not affected by propertyNames",
+        "schema": {
+            "$schema": "https://json-schema.org/draft/next/schema",
+            "propertyNames": {"maxLength": 1},
```

Comment:
formatting?

Reply:
do you mean `{"maxLength": 1}`. I see such compact formatting used in other places in tests in same file also

Reply:
Ah, that seems true. I don't have objections then.

Reply:
Unfortunately, the code style in the tests is not consistent. We really should at some point agree on a code style and enforce it.